src/constants/v3Farms.ts

```diff
@@ -23,10 +38,40 @@ export const DpiEthRewards: V3Farm = {
       startTime: 1629493200,
       endTime: 1630789200,
       refundee: '0x9467cfADC9DE245010dF95Ec6a585A506A8ad5FC',
+      dateText: 'August 20th, 2021 - September 4th, 2021',
+      img: {
+        alt: 'DPI Icon',
+        src: 'https://set-core.s3.amazonaws.com/img/social_trader_set_icons/defi_pulse_index_set.svg',
+      },
     },
   ],
 }
 
-const farms = [DpiEthRewards]
+export const MviEthRewards: V3Farm = {
+  tokenPair: 'MVI/ETH',
+  pool: '0x9359c87B38DD25192c5f2b07b351ac91C90E6ca7',
```

Comment:
https://info.uniswap.org/#/pools/0x9359c87b38dd25192c5f2b07b351ac91c90e6ca7 looks like DPI Eth pool. 

Should this be https://info.uniswap.org/#/pools/0x06ada8f74d99c6c200672b02e5c3341866ca3bfb instead?
```suggestion
  pool: '0x06ada8f74d99c6c200672b02e5c3341866ca3bfb',
```